lib/pangea/widgets/chat/message_toolbar.dart

```diff
@@ -147,14 +94,31 @@ class ToolbarDisplayController {
     if (toolbar == null) return;
     try {
       overlayEntry = Column(
-        mainAxisSize: MainAxisSize.min,
-        crossAxisAlignment: pangeaMessageEvent.ownMessage
-            ? CrossAxisAlignment.end
-            : CrossAxisAlignment.start,
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        mainAxisSize: MainAxisSize.max,
         children: [
-          toolbarUp ? toolbar! : overlayMessage,
-          const SizedBox(height: 6),
-          toolbarUp ? overlayMessage : toolbar!,
+          OverlayHeader(
```

Comment:
I think it's be nice to move these into a separate file / widget, called something like message_selection_overlay. It functions as an overlay over the whole chat page, so I think it makes sense to give it its own widget.

Reply:
Line 96-121